sklearn/metrics/_classification.py

```diff
@@ -713,6 +729,15 @@ class labels [2]_.
         else:
             w_mat = (w_mat - w_mat.T) ** 2
 
+    if np.sum(w_mat * expected) == 0:
```

Comment:
I think we should add a comment to explain why this would be equivalent to a ZeroDivisionError raised.